NASA-AMMOS/aerie-ui
# [AERIE-1955] feat: add datepicker component

This resolves https://jira.jpl.nasa.gov/browse/AERIE-1955.

To test:
1. Create a new plan
2. Verify that clicking on start/end times of the plan form opens the datepicker
3. Verify that clicking away from the datepicker or pressing the ESC key closes it
4. Verify that clicking on a day in the datepicker populates the input with the correct DOY string
5. Verify that typing in the date manually still works
6. After creating a new plan, add an activity
7. Set a start time for the activity via the activity's start time datepicker
8. Verify that it works the same
